specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -473,12 +600,55 @@ model TextCustomizedCategory {
   @maxLength(64)
   categoryName: string;
 
+  @doc("A predefined concept with its definition can be used directly in the subcategories.")
+  @added(Versions.v2023_10_30_Preview)
+  @removed(Versions.v2024_01_30_Preview)
   preDefinedConcepts?: PreDefinedConcept[];
-  subCategories: SubCategory[];
+
+  @doc("Subcategories are detailed classes of the category. It is required there is a class 'safe' with id=0.")
+  @renamedFrom(Versions.v2024_01_30_Preview, "subCategories")
+  classes: Class[];
```

Comment:
Given your description and related to my previous comment, why not keep this as `subcategories` (it's one word, not two) and call the model `Subcategory`? `Class` and `classes` won't be acceptable.

Reply:
We would have further discussion of naming internally, thanks for the information!
Moved custom category to next version.